pkg/etcd/etcd.go

```diff
@@ -180,7 +180,7 @@ func (e *Client) List(ctx context.Context, key string) (*Node, error) {
 		tailNode.Value = kv.Value
 	}
 
-	return root, nil
+	return root, resp.Header.Revision, nil
```

Comment:
is `Revision`  the revision of directory `key /`?

Reply:
is the newest revision under this path

Reply:
what's the reason to use it?

Reply:
we get all pump's information at time1, and then watch etcd event at time2, the event between time1 and time2 will lost. so need get the revision of list, and then watch etcd for this revision

Reply:
I review code, we use `watch` like this
```
revision, err := c.getPumpStatus(c.ctx) // always newest version
rch = c.EtcdRegistry.WatchNode(c.ctx, rootPath, revision)
```
is it different with you said?

Reply:
update in 46fe9f7